docs/MANUAL.md

```diff
@@ -298,16 +298,25 @@ TODO: screencap
 
 ## Sample Import Screen
 
-Accessible by hitting A,A on the “sample” parameter in the Instrument Screen.
-The samples of the library have to be located in a folder samplelib at the same level as the song folders (lgpt-xxxx). you can either put your samples in that directory or in sub-directories, allowing you to have some basic way of sorting the library. For example:
+Accessible by hitting A,A on the “sample:” parameter in the Instrument Screen.
+
+The samples of the library **must** to be located in a folder named `samplelib` at the top-level of the sdcard (lgpt-xxxx). You can either put your samples in that directory or in sub-directories of it, allowing you to have a basic way of sorting the library. 
+
+Note: subdirectories will be sorted before files, but otherwise the files will be listed in an undetermined order (ie. not necessarily alphabetical order).
+
+For example:
 
 TODO: screencap
 
 When entering the import screen, the current folder is the library root folder “samplelib”. All sample in that folder are listed.
-Use Up/Down to select a sample and 'A' to load it
-B+Left/Right to rotates between all sub directories.
+
+Use Up/Down to select a sample and the Play button to start/stop preview playback of the sample.
+RT+Play to import the currently selected sample and RT+Left to exit out of the Import Dialog back to the Instrument Screen.
```

Comment:
Following the logic for how RT and LT work in LGPT, shouldn't this be LT+Play? (i.e: LT would be used because Play is on the right hand side)

Reply:
Ah sorry, this is because I've been using the "M8" mapping for the last little while, which has LT & RT on opposite side to the std picotracker mapping. I'll change it to be `LT+Play` since this is original picotracker UI/behaviours anyways.